### ISSUE TITLE
[Bug]: sp-top-nav Focus colors incorrect in WHCM

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3387]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

ap-top-nav

### Expected behavior

Focus should use High Contrast colors and be visible

### Actual behavior

Text is not visible and border uses a non high contrast mode color

### Screenshots

<img width="1166" alt="Screenshot 2023-06-28 at 3 06 32 PM" src="https://github.com/adobe/spectrum-web-components/assets/1724479/bdc27885-400f-4f42-874f-4e365b236f26">

### What browsers are you seeing the problem in?

Chrome, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/top-nav/#page-4
2. Turn on WHCM or set forced-colors in Dev Tools
3. tab to the the top nav and observe

**Comment by Westbrook** [comment]
This is getting cleaned up in #3333 as we speak. Thanks @mlogsdon18!

**Comment by Westbrook** [comment]
fixed by https://github.com/adobe/spectrum-web-components/pull/3333
